reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
```

Comment:
If AUI zip is upgraded then the "old" zip won't be there, as each version is exclusive. So, the zip wouldn't be downloaded, but caused by the zip whole upgrade (name + md5) not by md5 only.